Formula/ghostscript.rb

```diff
@@ -47,25 +55,32 @@ class Ghostscript < Formula
     sha256 "0eb6f356119f2e49b2563210852e17f57f9dcc5755f350a69a46a0d641a0c401"
   end
 
-  # Fix build on ARM Big Sur, updating config.{guess,sub}
-  # https://bugs.ghostscript.com/show_bug.cgi?id=703095
-  patch do
-    url "https://raw.githubusercontent.com/Homebrew/formula-patches/98c39e09/ghostscript/config.patch"
-    sha256 "155cf2ee5a498d441c3194ba3d75cb7812beaa3f507a72017174a884bf742862"
-  end
-
   def install
-    on_linux do
-      # Fixes: ./soobj/dxmainc.o: file not recognized: File truncated
-      ENV.deparallelize
-    end
+    # Fix vendored tesseract build error: 'cstring' file not found
+    # Remove when possible to link to system tesseract
+    ENV.append_to_cflags "-stdlib=libc++" if ENV.compiler == :clang
```

Comment:
```suggestion
    ENV.libcxx
```
Exactly the same thing, but less verbose. (It even does the `:clang` check)

Reply:
Looks like `ENV.libcxx` fails for me.

The `./configure` output is identical, but the `make` output only has `-stdlib=libc++` if I use former.
Does the `ENV.libcxx` only modify C++ flags?

I think issue happens compiling C code, so need a way of adding stdlib into `CFLAGS`, if there is a simpler alternative available.

Reply:
Ah, you're right. `ENV.libcxx` only appends to `CXXFLAGS`. We can keep this as-is then.